lib/tessel/deploy.js

```diff
@@ -3,6 +3,7 @@ var path = require('path');
 var zlib = require('zlib');
 
 // Third Party Dependencies
+var acorn = require('acorn');
```

Comment:
FWIW, `acorn.parse` is significantly faster than the built-in uglifyjs2 parser ([by the author's own admission](http://lisperator.net/blog/uglifyjs-why-not-switching-to-spidermonkey-ast/)). Ideally, code will only have to go through `acorn.parse`, so it _should_ improve performance. Either way, I will put together some profiles.